overlord/devicestate/handlers_install.go

```diff
@@ -212,6 +235,12 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("cannot make run system bootable: %v", err)
 	}
 
+	// store install-mode log into ubuntu-data partition when
+	// persistent journal is selected
```

Comment:
this currently doesn't care whether persistent journal is selected or not, it always writes the logs

Reply:
Indeed, I updated it.